pkg/sql/sqlbase/system.go

```diff
@@ -123,6 +123,21 @@ CREATE TABLE system.jobs (
 	INDEX (status, created),
 	FAMILY (id, status, created, payload)
 );`
+
+	SessionsTableSchema = `
+CREATE TABLE system.sessions (
+	id             SERIAL     PRIMARY KEY,
+	"hashedSecret" BYTES      NOT NULL,
+	username       STRING     NOT NULL,
+	"createdAt"    TIMESTAMP  NOT NULL DEFAULT now(),
+	"expiresAt"    TIMESTAMP  NOT NULL,
+	"revokedAt"    TIMESTAMP,
+	"lastUsedAt"   TIMESTAMP  NOT NULL DEFAULT now(),
+	"auditInfo"    STRING,
+	INDEX("expiresAt"),
+	INDEX("createdAt"),
+	FAMILY(id, "hashedSecret", username, "createdAt", "expiresAt", "revokedAt", "lastUsedAt", "auditInfo")
```

Comment:
Now that we're using Postgres case-folding rules, seems like we should switch to Postgres-style snake_case identifiers so we don't have to quote every multi-word column name. 

There's a consistency argument here too, though. I don't feel particularly strongly.